stack.yaml

```diff
@@ -0,0 +1,21 @@
+resolver: lts-7.1
+
+packages:
+- '.'
+- location:
```

Comment:
If it doesn't work out, I'm fine with what you had.  Maybe it worked for me because I already had a successful build as a base, and then incremental built after editing stack.yaml.